app/frontend/components/taskList/taskRow.tsx

```diff
@@ -0,0 +1,58 @@
+import { ErrorMessage } from '@hookform/error-message'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { useForm } from 'react-hook-form'
+import { TaskFragment, useTaskUpdateMutation } from '../../generated/graphql'
+import { InputField } from '../inputField/inputField'
+import { CgSpinner } from 'react-icons/cg'
+import { TaskFormData, taskInputSchema } from './taskList'
+
+interface TaskRowProps {
+  task: TaskFragment
+}
+
+export const TaskRow = ({ task }: TaskRowProps) => {
+  const [{ fetching }, taskUpdate] = useTaskUpdateMutation()
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<TaskFormData>({
+    mode: 'onChange',
+    defaultValues: {
+      title: task.title,
+    },
+    resolver: yupResolver(taskInputSchema),
+  })
+
+  const handleSubmitTask = async (taskData: TaskFormData) => {
+    try {
+      const result = await taskUpdate({
+        id: task.id,
+        data: {
+          projectId: task.project.id,
+          title: taskData.title,
+        },
+      })
+      if (result.error) {
+        throw new Error(`GraphQL Error ${result.error}`)
+      }
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error)
+    }
+  }
+
+  return (
+    <div className="flex min-w-[500px]">
+      <InputField
+        variant="primary"
+        className="ml-2"
+        {...register('title', { required: true })}
+        onBlur={handleSubmit(handleSubmitTask)}
+      />
+
+      {fetching && <CgSpinner className="ml-2 inline h-8 w-8 animate-spin dark:text-blue-600" />}
+      <ErrorMessage errors={errors} name="title" as={<span aria-label="error field" className="ml-2 text-red-700" />} />
```

Comment:
@WINt1703, do you need the `aria-label` to make this span accessible in your test?
This is not what aria-label is for.

> The `aria-label` attribute defines a string value that labels an interactive element.
> https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

But you can use the `data-testid` attribute in testing library instead:
https://testing-library.com/docs/queries/bytestid/

Reply:
use role alert for this: (https://www.w3.org/WAI/GL/wiki/Using_ARIA_role_of_alert_for_Error_Feedback_in_Forms)